src/frontend/src/lib/i18n/en.json

```diff
@@ -240,7 +241,8 @@
 			"erc20_data_undefined": "Erc20 transaction Data cannot be undefined or null.",
 			"data_undefined": "Transaction Data cannot be undefined or null.",
 			"no_identity_calculate_fee": "No identity provided to calculate the fee for its principal.",
-			"invalid_address": "The address is invalid. Please try again with a valid address identifier."
+			"invalid_address": "The address is invalid. Please try again with a valid address identifier.",
```

Comment:
I would not use "address identifier" as it relates to old ICP address identifieir. Maybe "destination" or "wallet address" instead?

Reply:
Agreed!

Reply:
```json
"invalid_destination": "The destination is invalid. Please try again with a valid wallet address or destination."
```

what do you think?

Reply:
Good idea!